AdaSupport/langdetect
# ML-672: Limit Language Detection to Ada-Supported Languages

### Why was this PR opened? ###
The off-the-shelf language detection model predicts a suite of languages. Limit the model to detect only supported languages.

### What are the changes? ###
Removed unsupported language profiles and renamed existing profiles (`zh-cn` and `id`) to match the language code followed by `api` repo.

### How can someone test the changes? ###
```python setup.py install```

On a Python interpreter:
from langdetect import detect_langs

&#35;A sentence belonging to Afrikaans, Welsch etc should not be predicted here.
detect_langs("input sentence here")

__Example:__
detect_langs("你好吗") #should give you `zh` and not `zh-cn`

---
- [x] Code changes are documented
~- [ ] Unit tests cover code changes~
~- [ ] PR was tested on ada-dev~
~- [] Deployment plan is included in PR if required~
